src/LogViewer/index.js

```diff
@@ -75,7 +75,7 @@ export default class LogViewer extends React.Component {
       parent.postMessage(JSON.stringify(qs), '*');
     }
 
-    history.pushState(null, '', `${location.origin}${location.pathname}?${stringify(qs)}`);
+    history.replaceState(null, '', `${location.origin}${location.pathname}?${stringify(qs)}`);
```

Comment:
@eliperelman `window.history.length` stays the same when modifying the state and this is all we care about. I think it's good enough to merge :)